### DISCUSSION TITLE
Option for server host to disable text chat

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#7894] [Ideas and suggestions]
_Personally, I hate this idea. I just suggested it so that people stop whining about how they're better than me just because they never learnt how to read, and that text chat needs to be removed. This suggestion aims to both appease them and make them shut up._ 

Currently, voice chat is toggleable. Basically noone even uses that setting, but that's besides the point, text chat should also be toggleable. 

Why? There's a plethora of people who refuse to read. There's another plethora who reasonably don't want to speak, or can't, and being banned from a server just because you use text chat can put you in a bad mood, especially when it wasn't indicated in the server description.

**Comment by gerakis** [comment]
Ok that was pretty funny. Get a mic though. It might be weird the first few hours but after that it becomes second nature. Just power through man. I think its good for social skills. People get real mean when typing but when talking are nicer. The human side comes through more.

As for not being able to read clearly I can (or I wouldnt be writing this) but there is a lot going on in this game. Text chat gets lost by the wayside if things are exciting.

I would even argue for text to speech within the game. If you guys could still be audible even while typing that would be fantastic and the best of both worlds. If they could do the Steven Hawking or MLG voice that would be even better. This was the main reason Moonbase Alpha was popular.

I actually do like the original idea though. Just prevent anyone without voice chat enabled from joining the server if the host wants it that way. Prevent them from feeling mad from getting kicked, and prevents me from having to kick them. Thats a win win. It saves everyone time. From the number of game lobbies that say "mic or ban" I think there is a lot of people that feel this way.

**Reply by ghost** [reply]
TTS is a great idea, I cannot however get a mic because of background noise and IRL complications. I do not like using text chat, it is merely my only option.
